tests/plugins/tableselection/scrollmouseover.js

```diff
@@ -6,20 +6,21 @@
 	'use strict';
 
 	bender.editors = {
-		scrollable: {
+		classic: {
```

Comment:
What i meant here is to define two separate editors. One classic (basically same definition as it was before, but different name). And the other one, called `divarea` with `divarea` added to extraPlugins.